### ISSUE TITLE
Ready for review

### ISSUE THREAD

**Issue by nayankk** [xc0ffeelabs/taxicab#1]
Our Sprint 1 progress is now ready for review. /cc @codepathreview @codepath

**Comment by codepathreview** [comment]
:+1:  Nice work creating wire-frames and implementing few stories for sprint 1, but the implementation seems far behind where it should have been by now. Few notes to mention:
- It is important create tasks for each of your stories/features and have them assigned to team members in the form of issues. Please take care of this before you work on sprint 2.
- Consider diving a bigger parent task into a number of smaller subtasks - provides a better picture of the progress on the issue.
- Work load needs to be divided equally among all members of the team. This is critical in working as a team and also gives equal learning opportunity to everyone involved. Please make sure everyone [contributes](https://github.com/xc0ffeelabs/taxicab/graphs/contributors) to the project in the next sprint.
- Group projects can reinforce skills that are relevant to both group and individual work including the ability to plan and manage time. For this reason, it is imperative that you group all detailed issues into [milestones](https://github.com/xc0ffeelabs/taxicab/milestones) representing sprints so that its easier for you to track progress of the project.
- Good to see that your team created regular commits to sync up your code to avoid having multiple conflicts at the end.
- Consider creating tasks to QA the app a few times to check the basic flow. Integration testing will especially be important for this project since each team member is working on a separate module. The best part is seeing it all come together into a polished app.
- Consider creating group [milestones/pull requests](https://help.github.com/articles/creating-and-editing-milestones-for-issues-and-pull-requests/) to better track your progress and review each others code.

The implementation seems to be lagging behind on current timelines. Please take notes of these points and improve upon the same in the next sprint. Look forward to the rest of the functionality and improvement on the core flows by the end of the Sprint 2 so you can have enough time to add UI polish and advanced nice-to-have features later on. It is those little extra things you do that make everything feel more polished and desirable to use.

**Comment by nayankk** [comment]
Thanks a lot for such detailed review, much appreciated.

Regarding creation of tasks, we're using Trello project management tool for user story break-down or task/sub-task management. Before we started the development, we sat together and derived a handful of user stories from wireframes, then broke them to sub-tasks. All our user stories can be found in our Trello agile dashboard [here](https://trello.com/b/eeiqsYbE/taxicab). As we are using free version of Trello, it doesn't allow to share the board publicly. However you can still view this dashboard by adding yourself as member via this [link](https://trello.com/invite/taxicabteam/4623523c13fed5d9b05fd845c8026f0f). Please let us know if you are with this project management setup or expect us to move our tasks as github tickets.

Regarding the work-load, there are 3 repositories that we are contributing to - [Taxi Cab User app](https://github.com/xc0ffeelabs/taxicab), [Taxi Cab driver app](https://github.com/xc0ffeelabs/taxicab-driver), [Taxi cab backend](https://github.com/xc0ffeelabs/taxicab-server). Contribution [link](https://github.com/xc0ffeelabs/taxicab/graphs/contributors) you mentioned might not be accurate as it just tracks the commits made in user app. Currently, I'm working on the user app and partially on driver app, while Srikanth is working on the taxi back end implementation and also on driver app. We hope to switch roles in future.

Regarding the progress, yes, we are lagging behind. (We switched our project, thus lost close a week initially). However, we are hopeful to have all required user stories by the end of this sprint before Wednesday. 

Thanks again.

@codepathreview @codepath
